src/main/java/edu/cornell/kfs/fp/document/service/CuDisbursementVoucherTaxService.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.kfs.fp.document.service;
+
+public interface CuDisbursementVoucherTaxService {
+	
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId);
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
```

Comment:
Vendor is misspelled in method name; please correct that. Also, it might be clearer to rename the methods to something like "isForeignVendorAndTaxReviewIsRequired" for the first and "isForeignVendorAndTaxReviewIsNotRequired" for the second.